ParkingZoneApp.Tests/Services/ParkingSlotServiceTests.cs

```diff
@@ -181,5 +194,59 @@ public void GivenIdAndNumber_WhenIsUniqueNumberIsCalled_ThenReturnFalse()
             _parkingSlotRepositoryMock.VerifyNoOtherCalls();
         }
         #endregion
+
+        #region IsSlotAvailableForReservation
```

Comment:
also add member data with many date times (to test is it working correctly or not)